### DISCUSSION TITLE
Has the EmbeddedResourceLoader been ported?

### DISCUSSION THREAD

**Discussion by checkymander** [IronLanguages/ironpython3#1723] [Q&A]
In IronPython2 the ability to load the standard library into C# via embedded resource was possible via the EmbeddedResourceLoader. e.g. https://github.com/IronLanguages/ironpython3/issues/382

This code no longer works in IronPython3, has this functionality been ported/moved somewhere else or has it not been implemented?

**Comment by slozier** [comment]
I'm not sure what `EmbeddedResourceLoader` is, but the code in the example works fine for me...
```c#
using System.Collections.Generic;
using System.Reflection;

using IronPython.Hosting;
using IronPython.Modules;

var engine = Python.CreateEngine();
var sysScope = engine.GetSysModule();
IList<object> metaPath = sysScope.GetVariable("meta_path");
var importer = new ResourceMetaPathImporter(Assembly.GetExecutingAssembly(), "scripts.zip");
metaPath.Add(importer);
engine.Execute("import hello");
```

Where `scripts.zip` is embedded in my assembly and contains `hello.py`:
```py
print('hello world!')
```
